HandsOn/Group56/mappings/zonasVerdes.yml

```diff
@@ -0,0 +1,21 @@
+prefixes:
+  rr: http://www.w3.org/ns/r2rml#
+  rml: http://semweb.mmlab.be/ns/rml#
+  gtfs: http://vocab.gtfs.org/terms#
+  xsd: http://www.w3.org/2001/XMLSchema#
+
+mappings:
+  zonasVerdes:
+    sources:
+      - [ZonasVerdes-updated.csv~csv]
+    s: http://group56.org/zonas-verdes/$(Distrito)-$(Numero_de_archivo)
+    po:
+      - [a, ex:zonasVerdes]
+      - [gtfs:distrito, $(Distrito),xsd:string]
```

Comment:
why you are using the gtfs prefix? I think you have to change it by the one you've defined